README.md

```diff
@@ -2,17 +2,17 @@
 
 ## Project Description
 
-This repository is part of the overarching Catena-X project.
+This repository is part of the overarching Eclipse Tractus-X project.
 
 BPDM is an acronym for business partner data management.
-This project provides core services for querying, adding and changing business partner base information in the Catena-X landscape.
+This project provides core services for querying, adding and changing business partner base information in the CEclipse Tractus-X landscape.
```

Comment:
Typo :)

Reply:
little hint for your commit messages 
example
```` bash
chore(docs): add something
updates eclipse-tractusx/bpdm#118
```` 
for PR´s or issues so this will autmatically update your issue via github with your refrenced commit

Reply:
Fixed the typo and reference the issue by updates keyword